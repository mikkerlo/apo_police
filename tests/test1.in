Безенов Данил Валерьевич Жданова Татьяна Александровна Гаврищук Никита Денисович Кожина Ольга Михайловна Накапкин Александр Денисович Непутяхин Алексей Дмитриевич Носач Артемий Александрович Стрелкова Софья Вячеславовна Токмаков Александр Викторович Фотеева Елена Вячеславовна Хохрякова Алена Дмитриевна Якушева Полина Владимировна Аверченкова Марися Сергеевна Алпатов Павел Станиславович Ахмедов Николай Тимурович Бабаев Алипаша Алияр Оглы Барышев Артем Владимирович Бекасов Максим Анатольевич Белевцева Елена Олеговна Березовская Леонилла Викторовна Бобошина Светлана Борисовна Богомолов Михаил Сергеевич Борзенков Александр Владимирович Бородич Константин Андреевич Булгаков Александр Дмитриевич Бусыгин Сергей Андреевич Бучнев Алексей Станиславович Бушуева Алина Павловна Быкова Екатерина Алексеевна Былинкина Анастасия Сергеевна Верёвкин Пётр Александрович Веселко Никита Игоревич Виноградов Олег Дмитриевич Вишневецкий Кирилл Сергеевич Воловинский Иннокентий Владимирович Воробьев Максим Игоревич Гаврилова Юлия Николаевна Гай Вячеслав Евгеньевич Гильманов Владислав Раулевич Голованов Сергей Александрович Григорьева Анастасия Владимировна Грунина Ирина Сергеевна Дворяткина Ксения Валерьевна Демидова Дарья Александровна Демичева Наталья Алексеевна Диких Варвара Сергеевна Дорошенков Михаил Михайлович Иванова Александра Максимовна Иванова Екатерина Сергеевна Калугина Анастасия Евгеньевна Караваев Максим Владимирович Клепиков Илья Александрович Кобяков Лев Викторович Комаров Михаил Игоревич Кораблев Владислав Алексеевич Кошелев Антон Андреевич Лавриков Виктор Викторович Ладилова Надежда Анатольевна Лебедева Алла Александровна Луцишина Ирина Валерьевна Малков Виктор Александрович Мапельман Валентина Михайловна Миллер Евгения Олеговна Михайлова Ирина Игоревна Морозова Полина Алексеевна Мусин Султан Астахович Нагорная Мария Александровна Назаров Роман Львович Некрасов Василий Алексеевич Нечагина Светлана Михайловна Нечепуренко Александра Ивановна Никифорова Полина Алексеевна Новиков Петр Алексеевич Новоселова Полина Владимировна Носова Елена Павловна Огаркина Алёна Андреевна Омельченко Максим Павлович Омельченко Юлия Георгиевна Павловская Вера Вениаминовна Панин Никита Алексеевич Пинигина Маргарита Евгеньевна Погарская Анастасия Владиславовна Подрезов Михаил Андреевич Пойманов Юрий Михайлович Покровская Софья Владимировна Поляченко Андрей Евгеньевич Пылина Анна Алексеевна Раджабов Камиль Розманович Родионова Римма Владимировна Рюмин Константин Данилович Садыгов Эльшан Гусейнага Оглы Самсонова Анна Денисовна Сатунин Николай Николаевич Семенов Михаил Александрович Смирнова Марина Сергеевна Степанова Марина Александровна Стрелкова Людмила Михайловна Стрельцов Дмитрий Сергеевич Субботина Полина Ильинична Сюняев Дамир Рафаилевич Терещук Олег Александрович Тихомирова Марина Ивановна Тищенко Роман Эдуардович Ткаченко Дарья Сергеевна Толстоган Анна Дмитриевна Толстолужинская Анастасия Евгеньевна Топников Михаил Александрович Троицкий Павел Александрович Трофимов Петр Алексеевич Трошин Сергей Дмитриевич Тюренков Илья Олегович Умарова Анастасия Мукимовна Устименко Ольга Олеговна Фадеев Михаил Андреевич Филатов Кирилл Андреевич Халайджи Александр Константинович Харитонов Даниил Андреевич Хегай Ксения Александровна Хораш Ольга Валерьевна Чашникова Анастасия Александровна Черевик Мария Александровна Чернецкий Федор Михайлович Черных Михаил Александрович Шендановин Юрий Сергеевич Шибанкова Екатерина Юрьевна
