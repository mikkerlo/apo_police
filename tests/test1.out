Аверченкова Марися Сергеевна
Алпатов Павел Станиславович
Ахмедов Николай Тимурович
Бабаев Алипаша Алияр оглы
Барышев Артем Владимирович
Безенов Данил Валерьевич
Бекасов Максим Анатольевич
Белевцева Елена Олеговна
Березовская Леонилла Викторовна
Бобошина Светлана Борисовна
Богомолов Михаил Сергеевич
Борзенков Александр Владимирович
Бородич Константин Андреевич
Булгаков Александр Дмитриевич
Бусыгин Сергей Андреевич
Бучнев Алексей Станиславович
Бушуева Алина Павловна
Быкова Екатерина Алексеевна
Былинкина Анастасия Сергеевна
Верёвкин Пётр Александрович
Веселко Никита Игоревич
Виноградов Олег Дмитриевич
Вишневецкий Кирилл Сергеевич
Воловинский Иннокентий Владимирович
Воробьев Максим Игоревич
Гаврилова Юлия Николаевна
Гаврищук Никита Денисович
Гай Вячеслав Евгеньевич
Гильманов Владислав Раулевич
Голованов Сергей Александрович
Григорьева Анастасия Владимировна
Грунина Ирина Сергеевна
Дворяткина Ксения Валерьевна
Демидова Дарья Александровна
Демичева Наталья Алексеевна
Диких Варвара Сергеевна
Дорошенков Михаил Михайлович
Жданова Татьяна Александровна
Иванова Александра Максимовна
Иванова Екатерина Сергеевна
Калугина Анастасия Евгеньевна
Караваев Максим Владимирович
Клепиков Илья Александрович
Кобяков Лев Викторович
Кожина Ольга Михайловна
Комаров Михаил Игоревич
Кораблев Владислав Алексеевич
Кошелев Антон Андреевич
Лавриков Виктор Викторович
Ладилова Надежда Анатольевна
Лебедева Алла Александровна
Луцишина Ирина Валерьевна
Малков Виктор Александрович
Мапельман Валентина Михайловна
Миллер Евгения Олеговна
Михайлова Ирина Игоревна
Морозова Полина Алексеевна
Мусин Султан Астахович
Нагорная Мария Александровна
Назаров Роман Львович
Накапкин Александр Денисович
Некрасов Василий Алексеевич
Непутяхин Алексей Дмитриевич
Нечагина Светлана Михайловна
Нечепуренко Александра Ивановна
Никифорова Полина Алексеевна
Новиков Петр Алексеевич
Новоселова Полина Владимировна
Носач Артемий Александрович
Носова Елена Павловна
Огаркина Алёна Андреевна
Омельченко Максим Павлович
Омельченко Юлия Георгиевна
Павловская Вера Вениаминовна
Панин Никита Алексеевич
Пинигина Маргарита Евгеньевна
Погарская Анастасия Владиславовна
Подрезов Михаил Андреевич
Пойманов Юрий Михайлович
Покровская Софья Владимировна
Поляченко Андрей Евгеньевич
Пылина Анна Алексеевна
Раджабов Камиль Розманович
Родионова Римма Владимировна
Рюмин Константин Данилович
Садыгов Эльшан Гусейнага оглы
Самсонова Анна Денисовна
Сатунин Николай Николаевич
Семенов Михаил Александрович
Смирнова Марина Сергеевна
Степанова Марина Александровна
Стрелкова Людмила Михайловна
Стрелкова Софья Вячеславовна
Стрельцов Дмитрий Сергеевич
Субботина Полина Ильинична
Сюняев Дамир Рафаилевич
Терещук Олег Александрович
Тихомирова Марина Ивановна
Тищенко Роман Эдуардович
Ткаченко Дарья Сергеевна
Токмаков Александр Викторович
Толстоган Анна Дмитриевна
Толстолужинская Анастасия Евгеньевна
Топников Михаил Александрович
Троицкий Павел Александрович
Трофимов Петр Алексеевич
Трошин Сергей Дмитриевич
Тюренков Илья Олегович
Умарова Анастасия Мукимовна
Устименко Ольга Олеговна
Фадеев Михаил Андреевич
Филатов Кирилл Андреевич
Фотеева Елена Вячеславовна
Халайджи Александр Константинович
Харитонов Даниил Андреевич
Хегай Ксения Александровна
Хораш Ольга Валерьевна
Хохрякова Алена Дмитриевна
Чашникова Анастасия Александровна
Черевик Мария Александровна
Чернецкий Федор Михайлович
Черных Михаил Александрович
Шендановин Юрий Сергеевич
Шибанкова Екатерина Юрьевна
Якушева Полина Владимировна
